tuxemon/core/components/menu/__init__.py

```diff
@@ -988,14 +989,15 @@ def draw_textItem(self, textlist, columns=1, pos_x=0, pos_y=0, align="middle", a
             self.menudis_y += line_spacing + longest_item.get_height()
 
 
-    def get_event(self, event, game=None):
```

Comment:
Disambiguation. Multiple other classes use the name get_event for functions that retrieve conditional variables from a list or dict. Events are pushed to the menu class from a parent rather than retrieved by the menu's get_event (now menu_event). It can be changed back if necessary. The get_menu_event function in core/tools.py acts as the event retrieval and central source of control for menu's, eliminating redundant code present in each menu. Only new menu's have been modified to use this "menu controller" to ensure backwards compatibility and prevent the PR from going beyond the scope of it's intention. Each menu will now only contain use-case specific code in the menu_event function.